File: wlxx-services/wlxx-system/wlxx-system-client/src/main/java/com/xzg/wlxx/system/client/entity/po/EmpPo.java
package com.xzg.wlxx.system.client.entity.po;


import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import com.xzg.wlxx.common.base.BasePo;
import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;
import lombok.EqualsAndHashCode;

@TableName("emp")
@Data
@EqualsAndHashCode(callSuper = true)
public class EmpPo extends BasePo<EmpPo> {

    @TableField("emp_code")
    @NotBlank(message = "不能为空")
    private String empCode;

    @TableField("emp_name")
    @NotBlank(message = "不能为空")
    private String empName;

    private Long orgId;

    @Schema(description = "是否禁用", defaultValue = "0")
    private Boolean disabled;

}

File: wlxx-services/wlxx-system/wlxx-system-client/src/main/kotlin/com/xzg/wlxx/system/client/entity/po/EmpPo.kt
package com.xzg.wlxx.system.client.entity.po

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableName
import com.xzg.wlxx.common.base.BasePo
import lombok.Data
import lombok.EqualsAndHashCode

@TableName("emp")
@Data
@EqualsAndHashCode(callSuper = true)
data class EmpPo(
    @TableField("emp_code")
    var empCode: String? = null,

    @TableField("emp_name")
    var empName: String? = null,
    var orgId: Long? = null
) : BasePo<EmpPo?>()
